cmd/edge-orchestration/main.go

```diff
@@ -100,6 +101,7 @@ func orchestrationInit() error {
 
 	secure := os.Getenv("SECURE")
 	mnedc := os.Getenv("MNEDC")
+	ui := os.Getenv("WEBUI")
```

Comment:
In Home Edge there are many devices which are very low resource device. Say like motion sensor/Camera which doesnt have display on its own(within the device), having a web interface listing resources/services might not be used in that device. Hence how about making WebUI run as separate container(Process). And only devices with good resources(Display&Computing) can run the WebUI

Reply:
@suresh-lc @tdrozdovsky Thanks for the great idea! 
I like the idea that run containers separately. However bolt db for resource, service, etc in the Edge Orchestration container is not exposed. As a result, there must be a web ui server for the interfaces at Edge Orchestration container at the moment. We can run containers as your idea later, if we have any db container like MariaDB.